src/components/templates/top/index.tsx

```diff
@@ -0,0 +1,35 @@
+import { useQuery } from 'react-query';
+import { Category, fetchCategories } from './fetcher';
+import { SimpleCard } from '../../molecules/SimpleCard';
+import { Headline } from '../../atoms/Headline';
+import styles from './styles.module.scss';
+
+export function Top() {
+  const { data, isLoading, error } = useQuery<Category[]>(
+    'categories',
+    fetchCategories,
+  );
+
+  if (isLoading) {
+    return <>Loading...</>;
+  }
+
+  if (error) {
+    return <>An error has occurred: {error}</>;
+  }
+
+  return (
+    <div className={styles.category_container}>
+      <Headline label="カテゴリ" headlineTypes="small" />
```

Comment:
↓みたいに`headlineTypes`を変えて、h2要素として表示するのが良さそうかなと思いました！
```suggestion
      <Headline label="カテゴリ" headlineTypes="middle" />
```

Reply:
もしかしてですけど、headlineTypesいらないですかね？
フォントサイズの大きさの変更はstyles.module.scss の中のheadlineでしてるのでh1 h2 h3 の違いもカバーしてるのかなと？